pkg/virtctl/vm/vm.go

```diff
@@ -389,9 +434,93 @@ func getDryRunOption(dryRun bool) []string {
 	return nil
 }
 
-func (o *Command) Run(args []string) error {
+func expandArgs() cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if (cmd.Flags().Changed(filePathArg) || cmd.Flags().Changed(filePathArgShort)) && len(args) > 0 {
```

Comment:
Isn't this handled by cobra directly? I mean you should be able to specify that you needs to use the flag only one either in the expanded or short version